### PR TITLE
Added breaking change alert for evcc

### PR DESCRIPTION
**Pull request by MikeTheTux**
based on:

- https://github.com/openhab/openhab-addons/pull/16114
- https://github.com/openhab/openhab-addons/pull/16428

### CODE REVIEW

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -133,6 +133,7 @@ ALERT;Twitter Binding has become X. All related references to Twitter and Tweets
 [4.2.0]
 ALERT;AndroidTV Binding: The thing configuration of 'port' has been renamed and split to 'googletvPort' and 'shieldtvPort'
 ALERT;ISM8 Binding: Most channels have changed and are now using Units of Measurements. Items must be adapted and the things must be recreated.
+ALERT;evcc Binding: Update to evcc API version 0.123.1 results in several new or updated channels. Existing Items will need to be adjusted.
```
Comment (holgerfriedrich) [inline distributions/openhab/src/main/resources/bin/update.lst:136]:
```suggestion
ALERT;evcc Binding: Update to evcc API version 0.123.1 results in several new or updated channels. Existing Items will need to be adjusted.
ALERT;ISM8 Binding: Most channels have changed and are now using Units of Measurements. Items must be adapted and the things must be recreated.
```

**Review by holgerfriedrich — COMMENTED** [review]
I think it should be in alphabetic order (not case sensitive).

File: distributions/openhab/src/main/resources/bin/update.lst (hunk 1 above)
Comment (MikeTheTux) [reply distributions/openhab/src/main/resources/bin/update.lst]: fixed

**Review by holgerfriedrich — APPROVED** [review]
LGTM

**Review by wborn — APPROVED** [review]
Thanks both!

**Pull request merged**